src/components/modal/Modal.scss

```diff
@@ -5,7 +5,6 @@
 /**************************************
  * Modal
  **************************************/
-
```

Comment:
nit: there's a removed line break but it's minor and can be merged as-is so only address this if you end up making a fix for something else.